settings.gradle.kts

```diff
@@ -1,12 +1,37 @@
+pluginManagement {
+  repositories {
+    mavenCentral()
+    gradlePluginPortal()
+  }
+}
+
 plugins {
-  `gradle-enterprise`
+  id("com.gradle.develocity") version "3.17.4"
 }
 
-gradleEnterprise {
+dependencyResolutionManagement {
+  repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+  repositories {
+    mavenCentral()
+    maven(url = "https://s3-us-west-2.amazonaws.com/dynamodb-local/release")
+  }
+}
+
+develocity {
   buildScan {
-    termsOfServiceUrl = "https://gradle.com/terms-of-service"
-    termsOfServiceAgree = "yes"
+    publishing {
+      termsOfUseUrl = "https://gradle.com/terms-of-service"
+      termsOfUseAgree = "yes"
+    }
+  }
+}
+
+gradle.lifecycle.beforeProject {
+  group = when {
+    path.startsWith(":wisp") -> "app.cash.wisp"
+    else -> "com.squareup.misk"
   }
+  version = findProperty("VERSION_NAME") as? String ?: "0.0-SNAPSHOT"
 }
```

Comment:
New API in Gradle 8.8!

Reply:
This is very cool

Reply:
tbh this is my first time using it and I'm not sure of all the implications -- hoping it is as good as it is ergonomic! This is meant to be isolated-projects-safe.